pkgs/development/ocaml-modules/unisim_archisec/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildDunePackage, fetchFromGitHub, ... }:
+
+buildDunePackage rec {
+  pname = "unisim_archisec";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "binsec";
+    repo = "unisim_archisec";
+    rev = "${version}";
+    hash = "sha256-ngOSpqQXegN70WxOYXd9p8GmMc9ABVi6RPgas6gy7uk=";
+  };
```

Comment:
What about downloading the released source code from there: https://github.com/binsec/unisim_archisec/releases/download/0.0.5/unisim_archisec-0.0.5.tbz

Reply:
Sure, I will use `fetchTarball` then

Reply:
Please use `fetchurl` instead.